cmake/SetupBOUTThirdParty.cmake

```diff
@@ -190,9 +190,12 @@ endif()
 message(STATUS "NetCDF support: ${BOUT_USE_NETCDF}")
 set(BOUT_HAS_NETCDF ${BOUT_USE_NETCDF})
 
-option(BOUT_USE_ADIOS2 "Enable support for ADIOS output" ON)
+option(BOUT_USE_ADIOS2 "Enable support for ADIOS output" OFF)
```

Comment:
I would prefer if such feature would be automatically enabled if they are found.